command/agent/eval_endpoint_test.go

```diff
@@ -110,6 +112,135 @@ func TestHTTP_EvalPrefixList(t *testing.T) {
 	})
 }
 
+func TestHTTP_EvalsDelete(t *testing.T) {
+	ci.Parallel(t)
+
+	testCases := []struct {
+		testFn func()
+		name   string
+	}{
+		{
+			testFn: func() {
+				httpTest(t, nil, func(s *TestAgent) {
+
+					// Create an empty request object which doesn't contain any
+					// eval IDs.
+					deleteReq := api.EvalDeleteRequest{}
+					buf := encodeReq(&deleteReq)
+
+					// Generate the HTTP request.
+					req, err := http.NewRequest(http.MethodDelete, "/v1/evaluations", buf)
+					require.NoError(t, err)
+					respW := httptest.NewRecorder()
+
+					// Make the request and check the response.
+					obj, err := s.Server.EvalsRequest(respW, req)
+					require.Equal(t,
+						CodedError(http.StatusBadRequest, "request does not include any evaluation IDs"), err)
+					require.Nil(t, obj)
+				})
+			},
+			name: "too few eval IDs",
```

Comment:
Noted. I think I do it this way because of the process in which I build the tests which starts with what the inputs are. I have no doubt this is done in many places where I have written tests, so something I can look into changing moving forward.